src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -182,10 +190,11 @@ def create_privacy_request(
                     for masking_secret in masking_secrets:
                         privacy_request.cache_masking_secret(masking_secret)
 
-            PrivacyRequestRunner(
-                cache=cache,
-                privacy_request=privacy_request,
-            ).submit()
+            if not config.execution.REQUIRE_MANUAL_REQUEST_APPROVAL:
```

Comment:
It looks like changes to this setting require a server restart to take effect. Do we foresee a need to have this change take effect without restarting the server?

Reply:
good thought, in my opinion, it's not a concern given our current number of users, but others may have different opinions